src/bin/common/parse_command.ml

```diff
@@ -0,0 +1,1157 @@
+    let dv = Numbers.Q.to_string (get_fm_cross_limit ()) in
+    let doc = Format.sprintf
+        "Skip Fourier-Motzkin variables elimination steps that may produce \
+         a number of inequalities that is greater than the given limit. \
+         However, unit eliminations are always done." in
+    let docv = "VAL" in
+    Arg.(value & opt string dv & info ["fm-cross-limit"] ~docv ~docs ~doc) in
+
+  let timelimit_interpretation =
+    let doc = "Set the time limit to $(docv) seconds for model generation \
+               (not supported on Windows)." in
+    let docv = "SEC" in
+    Arg.(value & opt (some float) None &
+         info ["timelimit-interpretation"] ~docv ~docs ~doc) in
+
+  let steps_bound =
+    let doc = "Set the maximum number of steps." in
+    let docv = "STEPS" in
+    Arg.(value & opt int (get_steps_bound ()) &
+         info ["S"; "steps-bound"] ~docv ~doc) in
+
+  let timelimit =
+    let doc =
+      "Set the time limit to $(docv) seconds (not supported on Windows)." in
+    let docv = "VAL" in
+    Arg.(value & opt (some float) None & info ["t"; "timelimit"] ~docv ~doc) in
+
+  let timelimit_per_goal =
+    let doc =
+      "Set the given timelimit for each goal, in case of multiple goals per \
+       file. In this case, time spent in preprocessing is separated from \
+       resolution time. Not relevant for GUI-mode." in
+    Arg.(value & flag & info ["timelimit-per-goal"] ~docs ~doc) in
+
+  Term.(ret (const mk_limit_opt $
+             age_bound $ fm_cross_limit $ timelimit_interpretation $
+             steps_bound $ timelimit $ timelimit_per_goal
+            ))
+
+let parse_output_opt =
+
+  let docs = s_output in
+
+  let interpretation =
+    let doc =
+      "Experimental support for counter-example generation. Possible \
+       values are 1, 2, or 3 to compute an interpretation before returning \
+       Unknown, before instantiation, or before every decision or \
+       instantiation. A negative value (-1, -2, or -3) will disable \
+       interpretation display. Note that $(b, --max-split) limitation will \
+       be ignored in model generation phase." in
+    let docv = "VAL" in
+    Arg.(value & opt int (get_interpretation ()) &
+         info ["interpretation"] ~docv ~docs ~doc) in
+
+  let model =
+    let doc = Format.sprintf
+        "Experimental support for models on labeled terms. \
+         $(docv) must be %s. %s shows a complete model and %s shows \
+         all models."
+        (Arg.doc_alts ["none"; "default"; "complete"; "all"])
+        (Arg.doc_quote "complete") (Arg.doc_quote "all") in
+    let docv = "VAL" in
+    Arg.(value & opt model_conv MNone & info ["m"; "model"] ~docv ~doc) in
+
+  let unsat_core =
+    let doc = "Experimental support for unsat-cores." in
+    Arg.(value & flag & info ["u"; "unsat-core"] ~doc) in
+
+  let output_format =
+    let doc =
+      Format.sprintf
+        "Answer unsat/sat/unknown instead of Valid/Invalid/I don't know. \
```

Comment:
```suggestion
      "Control the output format of the solver, $(docv) must be %s.
        The alt-ergo native format outputs Valid/Invalid/I don't know.
        The smtlib format outputs unsat/sat/unknown.
        If left unspecified, Alt-Ergo will use its heuristics to determine the
        adequate output format according to the input format."
```
Multi-line suggestions do not seem to work very-well, you might have to manually change this to accept the suggestion, :/